test-sample-test.ts

```
@@ -0,0 +1,148 @@
+// Sample TypeScript file for testing AI Code Review Action
+// This file contains various code issues that AI should catch
+
+import * as fs from 'fs';
+import { promisify } from 'util';
+
+// Issue 1: Hardcoded API key (security vulnerability)
+const API_KEY = 'sk-1234567890abcdef';
```

ESLint

[error] 'API_KEY' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)